ITR/data/data_warehouse.py

```diff
@@ -2,12 +2,12 @@
 import logging
 import pandas as pd
 import numpy as np
-from uncertainties import unumpy as unp
 
 from abc import ABC
 from typing import List, Type
 from pydantic import ValidationError
 
+import ITR
```

Comment:
Unfortunately for me, these changed are still not enough, to make the tool work w/o custom `pint` module.
When i execute the newly added jupiter notebook `examples/co2budget.ipynb`, it throws an exception in the 2nd cell, when trying to `import ITR`

~~~~
---------------------------------------------------------------------------
UndefinedUnitError                        Traceback (most recent call last)
Input In [2], in <cell line: 1>()
----> 1 import ITR
      2 from ITR.data.base_providers import BaseProviderProductionBenchmark, BaseProviderIntensityBenchmark
      3 from ITR.interfaces import ETimeFrames, EScope, IProductionBenchmarkScopes, IEIBenchmarkScopes

File ~/os-climate/ITR/ITR/__init__.py:5, in <module>
      1 """
      2 This package helps companies and financial institutions to assess the temperature alignment of investment and lending
      3 portfolios.
      4 """
----> 5 from .data import osc_units
      6 from . import data
      7 from . import utils

File ~/os-climate/ITR/ITR/data/__init__.py:32, in <module>
     30 pint.Unit = ureg.Unit
     31 pint.Measurement = ureg.Measurement
---> 32 pint.Context = ureg.Context
     34 # FIXME: delay loading of pint_pandas until after we've initialized ourselves
     35 from pint_pandas import PintType

File ~/os-climate/ITR/itr_ui/lib64/python3.9/site-packages/pint/registry.py:330, in BaseRegistry.__getattr__(self, item)
    328 def __getattr__(self, item):
    329     getattr_maybe_raise(self, item)
--> 330     return self.Unit(item)

File ~/os-climate/ITR/itr_ui/lib64/python3.9/site-packages/pint/unit.py:47, in Unit.__init__(self, units)
     45     self._units = units
     46 elif isinstance(units, str):
---> 47     self._units = self._REGISTRY.parse_units(units)._units
     48 elif isinstance(units, Unit):
     49     self._units = units._units

File ~/os-climate/ITR/itr_ui/lib64/python3.9/site-packages/pint/registry.py:1161, in BaseRegistry.parse_units(self, input_string, as_delta, case_sensitive)
   1159 for p in self.preprocessors:
   1160     input_string = p(input_string)
-> 1161 units = self._parse_units(input_string, as_delta, case_sensitive)
   1162 return self.Unit(units)

File ~/os-climate/ITR/itr_ui/lib64/python3.9/site-packages/pint/registry.py:1391, in NonMultiplicativeRegistry._parse_units(self, input_string, as_delta, case_sensitive)
   1388 if as_delta is None:
   1389     as_delta = self.default_as_delta
-> 1391 return super()._parse_units(input_string, as_delta, case_sensitive)

File ~/os-climate/ITR/itr_ui/lib64/python3.9/site-packages/pint/registry.py:1189, in BaseRegistry._parse_units(self, input_string, as_delta, case_sensitive)
   1187 many = len(units) > 1
   1188 for name in units:
-> 1189     cname = self.get_name(name, case_sensitive=case_sensitive)
   1190     value = units[name]
   1191     if not cname:

File ~/os-climate/ITR/itr_ui/lib64/python3.9/site-packages/pint/registry.py:683, in BaseRegistry.get_name(self, name_or_alias, case_sensitive)
    681 candidates = self.parse_unit_name(name_or_alias, case_sensitive)
    682 if not candidates:
--> 683     raise UndefinedUnitError(name_or_alias)
    684 elif len(candidates) == 1:
    685     prefix, unit_name, _ = candidates[0]

UndefinedUnitError: 'Context' is not defined in the unit registry
~~~~

Reply:
i could try to make it work for me, but for now, i think i will prioritize the "template-v2-only" split, which we discussed earlier

Reply:
Do you have the latest version of Pint installed: https://github.com/hgrecco/pint/releases/tag/0.20.1 ?

Reply:
I have a version of pint, which `requirements.py` specified `pip` to install for me.
I will try the latest version

Reply:
I have updated requirements.txt and tested that it works with standard-issue Pint, Pint-Pandas, and Pandas (without uncertainties).